fuzzers/utils.py

```diff
@@ -150,19 +161,44 @@ def get_dictionary_path(target_binary):
     return None
 
 
-def set_default_optimization_flag(env=None):
-    """Set default optimization flag if none is already set."""
-    if not env:
+def set_fuzz_target(env=None):
```

Comment:
I'm confused why this function is necessary now?

Reply:
Used by some fuzzers

aflcc/fuzzer.py:    fuzz_target = os.getenv('FUZZ_TARGET')
aflplusplus/fuzzer.py:        # For CmpLog build, set the OUT and FUZZ_TARGET environment
aflplusplus/fuzzer.py:        fuzz_target = os.getenv('FUZZ_TARGET')
aflplusplus/fuzzer.py:            new_env['FUZZ_TARGET'] = os.path.join(cmplog_build_directory,
klee/fuzzer.py:    fuzz_target = os.getenv('FUZZ_TARGET')

i removed the hack we had in dockerfile to parse it since we would be parsing type attribute anyway here.